feed/tests/test_workflowlevel1view.py

```diff
@@ -101,3 +239,32 @@ def test_delete_workflowlevel1_normal_user(self):
         response = view(request, pk=wflvl1.pk)
         self.assertEquals(response.status_code, 403)
         WorkflowLevel1.objects.get(pk=wflvl1.pk)
+
+    def test_delete_workflowlevel1_program_admin_just_one(self):
+        # Create a program
+        factory = APIRequestFactory()
+        data = {'name': 'Save the Pandas'}
+        request = factory.post('/api/workflowlevel1/', data)
+        request.user = self.tola_user.user
+        view = WorkflowLevel1ViewSet.as_view({'post': 'create'})
+        response = view(request)
+        first_program_id = response.data['id']
+
+        # Create another program
+        factory = APIRequestFactory()
+        data = {'name': 'Save the Children'}
+        request = factory.post('/api/workflowlevel1/', data)
+        request.user = self.tola_user.user
+        view = WorkflowLevel1ViewSet.as_view({'post': 'create'})
+        response = view(request)
+
+        # Delete only the latter program
+        factory = APIRequestFactory()
```

Comment:
You're setting the variable factory three times in this test case and you also set the factory variable in all the test cases. It would be better to create `self.factory` in the `setUp` and use it when we need/want.

Reply:
Das Stimmt! Done